cli/framework.py

```diff
@@ -19,6 +22,25 @@ def fetchUserPass(self, env):
             self.passw = settings.getPass(env)
         print("Using u:{}, p:****".format(self.user))
 
+    def app_id(self, template_file, framework):
```

Comment:
I disagree. That would be redundant. There isn't enough difference between the implementations to justify to separate methods. What's your take on it?